code/__HELPERS/_macros.dm

```diff
@@ -5,8 +5,30 @@
 // fun if you want to typecast humans/monkeys/etc without writing long path-filled lines.
 #define ishuman(A) istype(A, /mob/living/carbon/human)
 
+#define isjusthuman(A) A.type == /mob/living/carbon/human
+
 #define ismonkey(A) istype(A, /mob/living/carbon/monkey)
 
+#define isvox(A) istype(A, /mob/living/carbon/human/vox)
```

Comment:
Could I change the macros to instead check for ishuman(), then check the mob's species var to determine its race?